src/org/usfirst/frc/team694/robot/commands/BITCOINCheckCommand.java

```diff
@@ -7,7 +7,7 @@
 public class BITCOINCheckCommand extends ConditionalCommand { 
     
     public BITCOINCheckCommand() {
-        super(new BITCOINAutoCommand(), new BITCOINManualCommand());
+        super(new BITCOINAutoCommand(), new CrabArrowAcquireCommand());
```

Comment:
This change should not be made in the OI branch. You should talk to the people working on BITCOIN and tell them why this change is necessary, and have them make it, they know the code better and they should implement it themselves.